sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -352,6 +352,8 @@ class AccountStateView(
 
   def disableGasTracking(): Unit = trackedGasPool = None
 
+  def isGasTrackingEnabled(): Boolean = trackedGasPool.isDefined
```

Comment:
note: in the actual dev_evm, we moved `disableGasTracking` out of this class.
probably we don't need this new method as well.

Reply:
Removed.